### PR TITLE
docs: add docs for new tx-result

### PR DESCRIPTION
**Pull request by ryukato**
### What kind of change does this PR introduce? 
* [ ] bug fix
* [ ] feature
* [x] docs
* [ ] update

### What is the new behavior?
New docs for new tx-result

### CODE REVIEW

**Review by junbae2 — DISMISSED** [review]

File: docs/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+### Requirements
+* Node.js >= 10
+  * It uses ES2017.
+
+* [npm](https://www.npmjs.com/?target=_blank), preferably >=7
+
+Other dependencies are installed via npm(or yarn), and do not need to be pre-installed.
+
+### Install
+All the dependencies can be install via [npm](https://www.npmjs.com/?target=_blank) or [yarn](https://yarnpkg.com/?target=_blank)
```
Comment (yeojin-dev) [inline docs/docs/README.md]: IMO, `be installed` is better.

File: docs/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+* msgIndex: Index number of the message in a transaction
+* requestType: Type of the message. For example, "collection/MsgTransferNFT"
+* details: Value included in the message. Properties vary, depending on the type of the message. 
+  
+> Note
+>
+> `details` in each message isn't a fixed object, since it's hard to provide a concrete type of data when we support smart contract. With smart contracts, developers or owners of the smart contract can define their own messages with custom properties, which are not known to LINE Blockchain Developers.
+
+### txEvents
+Event refers to a change in status, triggered by a confirmed transaction. Events are distinguished as base coin related events, service token related events and item token related events. Refer to [LINE Blochain Docs](https://lbddocs-alpha.website.line-apps-dev.com/api-guide/Callback-Response#event) for event names and their properties for each type.
```
Comment (yeojin-dev) [inline docs/docs/README.md]: `[LINE Blochain Docs]` is typo.

File: docs/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+By they way, all events basically have `msgIndex` and `eventName` properties.
+* msgIndex: Index number of the message caused events.
+* eventName: Name of a event such as `EventCoinTransferred`, `EventTokenIssued` and so on.
```
Comment (yeojin-dev) [inline docs/docs/README.md]: `an event` is better.

**Review by yeojin-dev — COMMENTED** [review]
Please correct typos. :)

**Review by yeojin-dev — APPROVED** [review]

**Review by junbae2 — APPROVED** [review]

**Pull request merged**